gsoc2018-bharat/src/CheckPerson.py

```diff
@@ -0,0 +1,48 @@
+import os
```

Comment:
If this or other file has been copied and pasted from last years work, there's the question of how the repository should be organised. Should we fix the scripts from the other year , copy and paste, or re organise into a common structure ? @mommi84 might be the person to talk to about this.

Reply:
The best would be to reorganize the code into a common structure. Instead of a folder for each student, they could be named based on the projects, with an additional folder containing common files.